src/app/component/ProductActions/ProductActions.component.js

```diff
@@ -261,6 +267,32 @@ export default class ProductActions extends PureComponent {
         );
     }
 
+    renderCustomizableOptions() {
+        const {
+            product: { type_id, options },
+            getRequiredCustomizableOptions,
+            getSelectedCustomizableOptions
+        } = this.props;
+
+        if (type_id === CONFIGURABLE || isMobile.any()) {
```

Comment:
1) Actually that was intended to exclude only configurable products, but if necessary I can change that, for now, customizable options are rendered only for simple products
2) Customizable options on mobile are called in a different place (see `ProductPage.component`) because on the desktop they should appear above the "Add to Cart" button, but on mobile they should appear as expandable content in same place where is product information, reviews, etc.